### PR TITLE
Update COLLABORATION_SPACE_PROGRESSION.md

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: COLLABORATION_SPACE_PROGRESSION.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## I. Overview
 
-This governance policy describes how a new `Collaboration Space` can be established wthin the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [Collaboration Space Proposal Process](). It describes the [Stages]() a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.
+This governance policy describes how a new `Collaboration Space` can be established wthin the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [Collaboration Space Proposal Process](#collaborationspaceproposalprocess). It describes the [Stages](#stages) a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.
```
Comment (mhdawson) [inline COLLABORATION_SPACE_PROGRESSION.md:3]:
```suggestion
This governance policy describes how a new `Collaboration Space` can be established wthin the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [collaboration-space-proposal-process](#Collaboration Space Proposal Process). It describes the [Stages](#stages) a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.
```

File: COLLABORATION_SPACE_PROGRESSION.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## I. Overview
 
-This governance policy describes how a new `Collaboration Space` can be established wthin the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [Collaboration Space Proposal Process](). It describes the [Stages]() a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.
+This governance policy describes how a new `Collaboration Space` can be established wthin the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [collaboration-space-proposal-process](#Collaboration Space Proposal Process). It describes the [Stages](#stages) a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.
```
Comment (mhdawson) [inline COLLABORATION_SPACE_PROGRESSION.md:3]:
```suggestion
This governance policy describes how a new `Collaboration Space` can be established wthin the OpenJS Foundation, hereinafter referred to as the "Foundation", via the [Collaboration Space Proposal Process](#collaboration-space-proposal-process). It describes the [Stages](#stages) a collaboration space may be admitted under and what the criteria and expectations are for a given stage, as well as the acceptance criteria for a space to move from one stage to another.
```

**Review by ljharb — APPROVED** [review]

**Review by codebytere — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Review by eemeli — APPROVED** [review]

**Comment by mhdawson** [comment]
@openjs-foundation/cpc propossing fast-track since its just fixup versus changes.

**Comment by mhdawson** [comment]
Landing since I think its long since the time were it can land.

**Pull request merged**